app/my-strategies/page.tsx

```diff
@@ -40,10 +40,13 @@ export default function MyStrategies() {
 								<Table.Row
 									className="cursor-pointer"
 									key={i}
-									onClick={() => {
-										router.push(
-											`${window.location.origin}/my-strategies/${order.addEvents[0].transaction.id}-${order.network}`
-										);
+									onClick={(e) => {
```

Comment:
can we just use a normal `<a>` instead of reinventing all the individual behaviours of `<a>` in ad hoc js?

Reply:
with the <a> tag we have to use only one `Table.Cell` which makes things more complicated with the table component we have. we would have to position each column items with css

i think the current implementation is better in comparison to this

Reply:
i would really like to see it be <a> tags, why can't an <a> tag wrap a row? does the html spec actually prohibit that?